Loki CRDs missing on fresh install

### Steps to reproduce the problem

create new k8s cluster
cd infrastructure/helm/quantumserverless
helm install qs .

### What is the current behavior?

Install fails with an error about missing CRDs:

```
$ helm install qs .
coalesce.go:223: warning: destination for loki.gateway.affinity is a table. Ignoring non-table value (podAntiAffinity:
  requiredDuringSchedulingIgnoredDuringExecution:
    - labelSelector:
        matchLabels:
          {{- include "loki.gatewaySelectorLabels" . | nindent 10 }}
      topologyKey: kubernetes.io/hostname
)
Error: INSTALLATION FAILED: unable to build kubernetes objects from release manifest: [resource mapping not found for name: "loki" namespace: "" from "": no matches for kind "GrafanaAgent" in version "monitoring.grafana.com/v1alpha1"
ensure CRDs are installed first, resource mapping not found for name: "loki" namespace: "" from "": no matches for kind "LogsInstance" in version "monitoring.grafana.com/v1alpha1"
ensure CRDs are installed first, resource mapping not found for name: "loki" namespace: "" from "": no matches for kind "PodLogs" in version "monitoring.grafana.com/v1alpha1"
ensure CRDs are installed first]
```

### What is the expected behavior?

Install succeeds.

I _think_ this can be solved by tweaking some of the options in the helm chart...

--- a/infrastructure/helm/quantumserverless/values.yaml
+++ b/infrastructure/helm/quantumserverless/values.yaml
@@ -359,6 +359,7 @@ loki:
       type: ClusterIP
   monitoring:
     selfMonitoring:
+      enabled: false
       grafanaAgent:
         installOperator: false
   test:

